services/waffle/waffle-label.service.js

```diff
@@ -0,0 +1,104 @@
+'use strict'
+
+const Joi = require('joi')
+const { BaseJsonService } = require('..')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+
+const schema = Joi.array()
+  .items(
+    Joi.object({
+      label: Joi.object({
+        name: Joi.string().required(),
+        color: Joi.string()
+          .hex()
+          .required(),
+      }),
+      count: nonNegativeInteger,
+    })
+  )
+  .required()
```

Comment:
It looks like this schema is rejecting some valid cases. Here are a couple of valid API responses which are generating "invalid response data":

* https://api.waffle.io/waffleio/waffle.io/columns
* https://api.waffle.io/mysociety/fixmystreet/columns

I think we need to be more lenient on the inputs but maybe we also need to do some extra checks in `handle()` or `transform()`

Reply:
Good catch! It looks like there can apparently be labels that do not have a color, so I think a simple tweak to the schema should resolve that. 

I believe that means in the current/legacy implementation that the color would have been `undefined` in those use cases, so I wonder if we should keep that here or use the default color (78bdf2 apparently)?

Reply:
I've updated the schema accordingly, and the two above examples are working in the staging app now 👍 

![image](https://user-images.githubusercontent.com/13042488/53702474-f97f4100-3dcc-11e9-99ff-d44fea3f0809.png)


![image](https://user-images.githubusercontent.com/13042488/53702589-3ac42080-3dce-11e9-9a11-683401de085d.png)